### ISSUE TITLE
Videos / Images ordering

### ISSUE THREAD

**Issue by centrumkanalen** [signageos/smil-player#2]
Running the following
```
<seq repeatCount="indefinite">
    <video src="video1.mp4" />
    <img src="foo.jpg" />
    <video src="video1.mp4" />
</seq>
```
It plays video1.mp4 video2.mp4 foo.jpg (videos first, then images)
A workaround is to wrap each and every asset into <seq> markup, but a sleeker way would be preferable

**Comment by lukdanek** [comment]
We know about this issue, it's going to be addressed in the next release.

As a current workaround, wrap each media item to `<seq>`.

**Comment by timoteoponce** [comment]
So, the workaround should look like this, right?

```
```
<seq repeatCount="indefinite">
    <seq><video src="video1.mp4" /></seq>
    <seq><img src="foo.jpg" /></seq>
    <seq><video src="video1.mp4" /></seq>
</seq>

**Comment by lukdanek** [comment]
Hi, this issue was resolved in v1.3.0. With this version you no longer need the wrap items in seq
